src/sdk/db_sdk.cc

```diff
@@ -202,15 +202,15 @@ void ClusterSDK::CheckZk() {
 }
 
 bool ClusterSDK::Init() {
-    zk_client_ = new ::openmldb::zk::ZkClient(options_.zk_cluster, "", options_.session_timeout, "", options_.zk_path);
+    zk_client_ = new ::openmldb::zk::ZkClient(options_.zk_cluster, "", options_.zk_session_timeout, "", options_.zk_path);
     bool ok = zk_client_->Init();
     if (!ok) {
         LOG(WARNING) << "fail to init zk client with zk cluster " << options_.zk_cluster << " , zk path "
-                     << options_.zk_path << " and session timeout " << options_.session_timeout;
+                     << options_.zk_path << " and session timeout " << options_.zk_session_timeout;
         return false;
     }
     LOG(INFO) << "init zk client with zk cluster " << options_.zk_cluster << " , zk path " << options_.zk_path
-              << ",session timeout " << options_.session_timeout << " and session id " << zk_client_->GetSessionTerm();
+              << ",session timeout " << options_.zk_session_timeout << " and session id " << zk_client_->GetSessionTerm();
```

Comment:
please resolve the `cpplint` wannings

Reply:
What do I need to do, I don t know anything about C++ programming language..😅

Reply:
> please resolve the `cpplint` wannings

What do I need to do, I don t know anything about C++ programming language..😅

Reply:
sorry i was just trying to accomplish this task, i have now given up, thanks because the programming language i am using is not c++, i don't know enough about c++, thanks

Reply:
you switch the `File changed` tab to look at the message as below:
<img width="447" alt="WX20220610-165703@2x" src="https://user-images.githubusercontent.com/6183996/173030166-45a09808-b911-4213-b45f-b93b1c15d4c3.png">

Reply:
> you switch the `File changed` tab to look at the message as below: <img alt="WX20220610-165703@2x" width="447" src="https://user-images.githubusercontent.com/6183996/173030166-45a09808-b911-4213-b45f-b93b1c15d4c3.png">

I try to solve